apps/charterafrica/src/components/OpportunityCardList/OpportunityCardList.js

```diff
@@ -0,0 +1,115 @@
+import Link from "@/commons-ui/next/Link";
+import { Typography, Box, Grid } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import React from "react";
+
+import OpportunityCard from "../OpportunityCard";
+
+import { neutral } from "@/charterafrica/colors";
+
+const OpportunityCardList = React.forwardRef(function OpportunityCardList(
+  props,
+  ref
+) {
+  const { title, grants } = props;
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+  const isSmallDesktop = useMediaQuery(theme.breakpoints.only("md"));
+  return (
+    <Box ref={ref}>
+      <Grid
+        container
+        justifyContent="space-between"
+        sx={{
+          marginBottom: "40px",
+        }}
+      >
+        <Grid item>
+          <Typography
+            variant="h5SemiBold"
+            color={neutral[900]}
+            sx={{
+              textTransform: "capitalize",
+            }}
+          >
+            {title}
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Link
+            href="/grants"
```

Comment:
```suggestion
            href="/opportunities/grants"
```
... all of these are children of /opportunities & should also be passed in via props rather than hard-coded like this.